taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -71,14 +74,22 @@ class DaskVine(Manager):
     #                      to the manager.
     # @param resources     A dictionary with optional keys of cores, memory and disk (MB)
     #                      to set maximum resource usage per task.
+    # @param lib_resources A dictionary with optional keys of cores, memory and disk (MB)
```

Comment:
I think we want all these arguments to be in  `get`, and not in the definition of the DaskVine manager. Note that environment and resources for regular tasks are defined in `get`.

Reply:
For clarification, I think I put these arguments in the definition for the function `get` and not `DaskVine(manager)`

Reply:
Sorry for the noise, I think I saw the class definition and blindly assumed that the first method was __init__.